app/views/simplified-account/settings/worldpay-details/index.njk

```diff
@@ -0,0 +1,42 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Worldpay details
+{% endblock %}
+
+{% macro showTask(task) %}
+  <li class="govuk-task-list__item govuk-task-list__item--with-link">
+    <div class="govuk-task-list__name-and-hint">
+      <a class="govuk-link govuk-task-list__link govuk-link--no-visited-state" href="{{ task.href }}"
+         aria-describedby="{{ task.id }}">
+        {{ task.linkText }}
+      </a>
+    </div>
+    <div class="govuk-task-list__status" id="{{ task.id }}">
+      {% if value.complete == true %}
+        Completed
+      {% else %}
+        <strong class="govuk-tag govuk-tag--blue">
+          Not yet started
+        </strong>
+      {% endif %}
+    </div>
+  </li>
+{% endmacro %}
+
+{% block settingsContent %}
+  <h1 class="govuk-heading-l">Worldpay details</h1>
+
+  {% if incompleteTasks %}
+    <p class="govuk-body govuk-!-margin-bottom-6">
+      You need to link your Worldpay account to GOV.UK Pay.
+    </p>
+
+    <ul class="govuk-task-list">
```

Comment:
we should be using the [task list component](https://design-system.service.gov.uk/components/task-list/) now that DSv5 is live.

See this [PR](https://github.com/alphagov/pay-selfservice/pull/4368/files#diff-161e20a8f11d7d595e5ce3788e032bc62a5d6dcee408f23e067a5c3b101f18e2)